src/@hooks/useProjectMutation.tsx

```diff
@@ -6,7 +6,7 @@ import { usePresetBuildOptionStore } from "./usePresetBuildOptionStore";
 import { useAuth } from "./useAuth";
 import APIClient from "../@utils/api";
 
-import type { UpdateProjectOptions } from "../@types/api";
+import type { PatchProjectOption, PutProjectOption } from "../@types/api";
```

Comment:
```suggestion
import type { PatchProjectOptions, PutProjectOptions } from "../@types/api";
```

타입 명칭은 복수형으로 통일되었으면 좋겠습니당

Reply:
CreateProjectOptions나 다른 옵션들은 여러 개의 옵션을 보내지만 Put, Patch의 경우는 하나의 옵션만 보내는 상황이라 단수형으로 해놓았어요. 하나의 옵션만 보내는 상황에서도 복수형을 사용하면 통일성이 깨져보이지 않을까 해서요~